fraud/service.go

```diff
@@ -110,35 +145,112 @@ func (f *service) processIncoming(
 		f.pubsub.BlacklistPeer(from)
 		return pubsub.ValidationReject
 	}
-	log.Warnw("received fraud proof", "proofType", proof.Type(),
-		"height", proof.Height(),
-		"hash", hex.EncodeToString(extHeader.DAH.Hash()),
-		"from", from.String(),
-	)
-	msg.ValidatorData = proof
+
+	err = f.put(ctx, proof.Type(), hex.EncodeToString(proof.HeaderHash()), msg.Data)
+	if err != nil {
+		log.Error("error while adding a fraud proof to storage: ", err)
```

Comment:
```suggestion
		log.Errorw("failed to store fraud proof", "err", err)
```

lets prefer to use this format for logging ^